sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -4903,7 +5017,8 @@ private void SetNameFieldsIfNull()
             BlobRequestConditions conditions,
             bool async,
             RequestContext context,
-            string operationName = default)
+            string operationName = default,
+            BlobGetPropertiesOptions options = default)
```

Comment:
`GetPropertiesInternal()` should either take `BlobGetPropertiesOptions`, or `BlobRequestConditions` and `userPrincipalName`